dist/docker/debian/build_docker.sh

```diff
@@ -103,7 +103,7 @@ run bash -ec "cat /scylla_bashrc >> /etc/bash.bashrc"
 run mkdir -p /etc/supervisor.conf.d
 run mkdir -p /var/log/scylla
 run chown -R scylla:scylla /var/lib/scylla
-run sed -i -e 's/^SCYLLA_ARGS=".*"$/SCYLLA_ARGS="--log-to-syslog 0 --log-to-stdout 1 --default-log-level info --network-stack posix"/' /etc/default/scylla-server
+run sed -i -e 's/^SCYLLA_ARGS=".*"$/SCYLLA_ARGS="--log-to-syslog 0 --log-to-stdout 1 --network-stack posix"/' /etc/default/scylla-server
```

Comment:
1. I assume --log-to-syslog 0 --log-to-stdout 1 are OK to be kept here?
2. I assume (in a separate PR) we may want to change --network-stack

Reply:
> I assume --log-to-syslog 0 --log-to-stdout 1 are OK to be kept here?

containers should generally log to stdout - I am not agaist flexibility here in general but I don't see any need driving it at this point, possibly the defaults may be different

> I assume (in a separate PR) we may want to change --network-stack

possibly, I am not sure what the default is there

btw. all of these would be addressed by adjusting the CLI logic to just let the last option value win (modulo array flags)

and even better: define these defaults in a config file, not with flags. but I don't think all of them actually have a config counterpart